### ISSUE TITLE
[BUG] Alpine display bug

### ISSUE THREAD

**Issue by bitcoin-tools** [bitcoin-tools/nodebuilder#480]
**Describe the bug**
On Alpine:
```
#11 1.963 Checking for Bitcoin Core 26.0... not found.
#11 1.969   Installing build dependencies... *
#11 4.077 * If you need ICU with non-English locales and legacy charset support, install
#11 4.077 * package icu-data-full.
#11 4.077 *
#11 6.856 ok.
```

**Steps to Reproduce**

**Expected behavior**

**Screenshots (optional)**

**Desktop (please complete the following information):**
 - OS: [e.g. Debian]
 - Environment [e.g. bare metal, docker]
 - Release [e.g. 1.0.0]

**Additional context**

**Comment by bitcoin-tools** [comment]
Issue appears to be resolved:
```
#11 2.627 Checking for Bitcoin Core 27.0... not found.
#11 2.634   Ensuring compile dependencies... ok.
#11 13.78   Downloading Bitcoin source code... ok.
#11 16.05   Analyzing hardware confgiruation... ok.
```

**Comment by bitcoin-tools** [comment]
Duplicate of #450. Fixed in #451.
